src/main/java/org/folio/inventory/resources/MoveApi.java

```diff
@@ -103,41 +117,97 @@ private void updateItems(RoutingContext routingContext, WebContext context, List
         .map(CompletableFuture::join)
         .map(Item::getId)
         .collect(toList()))
-      .thenAccept(updatedItemIds -> {
-        List<String> nonUpdatedIds = ListUtils.subtract(itemIdsToUpdate, updatedItemIds);
-        HttpServerResponse response = routingContext.response();
-        if (nonUpdatedIds.isEmpty()) {
-          JsonResponse.successWithEmptyBody(response);
-        } else {
-          JsonResponse.successWithIds(response, nonUpdatedIds);
-        }
-      });
+      .thenAccept(updatedIds -> respond(routingContext, idsToUpdate, updatedIds));
   }
 
-  private OkapiHttpClient createHttpClient(RoutingContext routingContext, WebContext context) throws MalformedURLException {
-    return new OkapiHttpClient(client, context, exception -> ServerErrorResponse.internalError(routingContext.response(),
-        String.format("Failed to contact storage module: %s", exception.toString())));
+  private void moveHoldings(RoutingContext routingContext) {
+
+    WebContext context = new WebContext(routingContext);
+    JsonObject holdingsMoveJsonRequest = routingContext.getBodyAsJson();
+
+    Optional<ValidationError> validationError = holdingsMoveHasRequiredFields(holdingsMoveJsonRequest);
+    if (validationError.isPresent()) {
+      unprocessableEntity(routingContext.response(), validationError.get());
+    } else {
```

Comment:
```suggestion
    if (validationError.isPresent()) {
      unprocessableEntity(routingContext.response(), validationError.get());
      return;
    } 
```

Then we don't need else here.